app/src/dataobjects/Addon.php

```diff
@@ -2,49 +2,58 @@
 
 use Elastica\Document;
 use Elastica\Type\Mapping;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\ORM\FieldType\DBBoolean;
+use SilverStripe\Assets\Image;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\Controller;
+use SilverStripe\View\ArrayData;
+use SilverStripe\ORM\DataObject;
+
 
 /**
  * An add-on with one or more versions.
  */
 class Addon extends DataObject
 {
 
-    public static $db = array(
+    private static $db = array(
         'Name'              => 'Varchar(255)',
         'Description'       => 'Text',
         'Type'              => 'Varchar(100)',
         'Readme'            => 'HTMLText',
-        'Released'          => 'SS_Datetime',
+        'Released'          => DBDatetime::class,
         'Repository'        => 'Varchar(255)',
         'Downloads'         => 'Int',
         'DownloadsMonthly'  => 'Int',
         'Favers'            => 'Int',
-        'LastUpdated'       => 'SS_Datetime',
-        'LastBuilt'         => 'SS_Datetime',
-        'BuildQueued'       => 'Boolean',
+        'LastUpdated'       => DBDatetime::class,
+        'LastBuilt'         => DBDatetime::class,
+        'BuildQueued'       => DBBoolean::class,
         // Module rating information
         'Rating'            => 'Int',
         'RatingDetails'     => 'Text',
     );
 
-    public static $has_one = array(
+    private static $has_one = array(
         'Vendor' => 'AddonVendor',
     );
 
-    public static $has_many = array(
+    private static $has_many = array(
         'Versions' => 'AddonVersion'
     );
 
-    public static $many_many = array(
+    private static $many_many = array(
         'Keywords'           => 'AddonKeyword',
-        'Screenshots'        => 'Image',
+        'Screenshots'        => Image::class,
         'CompatibleVersions' => 'SilverStripeVersion'
```

Comment:
It would probably make sense to use `AddonKeyword::class` and `SilverStripeVersion::class` here for consistency, even though it wouldn't have a functional change right now - might help to make adding namespaces easier in future